plugins/inputs/redis_sentinel/redis_sentinel.go

```diff
@@ -301,12 +301,14 @@ func convertSentinelMastersOutput(
 
 	fields := make(map[string]interface{})
 
-	fields["has-quorum"] = 0
+	fields["has_quorum"] = 0
 	if quorumErr == nil {
-		fields["has-quorum"] = 1
+		fields["has_quorum"] = 1
 	}
 
 	for key, val := range master {
+		key = strings.ReplaceAll(key, "-", "_")
```

Comment:
Since `strings.ReplaceAll` wasn't introduced until go 1.12, use something else. There is `internal.SnakeCase` that you could use instead if you wish.